data/json/professions.json

```diff
@@ -68,7 +71,8 @@
                 "boots_winter",
                 "knit_scarf",
                 "pockknife",
-                "water_clean",
+                "energy_drink",
+                "chips3",
```

Comment:
I don't understand the purpose of changing the standard survivor from having a bottle of water to an energy drink and chips. A regular water bottle fits the theme (nothing notable) much better.